.devcontainer/devcontainer.json

```diff
@@ -1,43 +1,24 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
-    "name": "PreTeXt-Full",
-    "features": {
-          "ghcr.io/rocker-org/devcontainer-features/pandoc:1": {}
-      },
-      "forwardPorts": [
-          27377,
-          27378
-      ],
-      "portsAttributes": {
-          "27378": {
-              "label": "CodeChat",
-              "onAutoForward": "openPreview",
-              "requireLocalPort": true,
-              "elevateIfNeeded": true,
-              "protocol": "https"
-          }
-      },
-    // Use 'postCreateCommand' to run commands after the container is created.
-    "postCreateCommand": "sudo bash ./.devcontainer/postCreateCommand.sh",
-  
-      // Configure tool-specific properties.
-      "customizations": {
-          "codespaces": {
-              "openFiles": [
-                  "project.ptx",
-                  "source/main.ptx"
-              ]
-          },
-          "vscode": {
-              "settings": {
-                  "CodeChat.CodeChatServer.Command": "CodeChat_Server"
-              },
-              "extensions": [
-                  "CodeChat.codechat",
-                  "oscarlevin.pretext-tools"
-              ]
-          }
-      }
-  
-      // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
-      // "remoteUser": "root"
-  }
\ No newline at end of file
+	"name": "PreTeXt-CLI-dev",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.11",
```

Comment:
If we want to test with pretext-full, then we can just use the oscarlevin/pretext:full docker image.  That will simplify the post-create command.